.changeset/over-nine-thousand.md

```diff
@@ -0,0 +1,7 @@
+---
+"@comet/admin": minor
+---
+
+Add support for block styles for the rte translation
+
+
```

Comment:
```suggestion
---
"@comet/admin-rte": patch
---

Fix losing custom block types when converting between editor state and HTML
```

This is a bug fix, not a feature. Also, the change is in `@comet/admin-rte`, not `@comet/admin`.